File: src/main/org/nlogo/agent/LinkManager3D.java
package org.nlogo.agent;

public strictfp class LinkManager3D
    extends LinkManager {

  public LinkManager3D(World3D world3D) {
    super(world3D);
  }

  @Override
  Link newLink(World world, Turtle src, Turtle dest, AgentSet breed) {
    return new Link3D(world, src, dest, breed);
  }
}

File: src/main/org/nlogo/agent/LinkManager3D.scala
// (C) Uri Wilensky. https://github.com/NetLogo/NetLogo

package org.nlogo.agent

class LinkManager3D(world3D: World3D) extends LinkManager(world3D) {
  override def newLink(world: World, src: Turtle, dest: Turtle, breed: AgentSet): Link =
    new Link3D(world, src, dest, breed)
}
